src/tomodachi_testcontainers/pytest/fixtures/containers.py

```diff
@@ -10,7 +10,7 @@
 
 
 @pytest.fixture(scope="session")
-def tomodachi_image() -> Generator[DockerImage, None, None]:
+def testcontainers_docker_image() -> Generator[DockerImage, None, None]:
     if image_id := os.getenv("TOMODACHI_TESTCONTAINER_IMAGE_ID"):
         yield get_docker_image(image_id)
     else:
```

Comment:
This is the change.

The fixture has nothing to do with [Tomodachi](https://github.com/kalaspuff/tomodachi), and it's renamed to allow the library to be used for testing services running on other frameworks.

Quite a breaking change though.